geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/stats/BinnedArray.scala

```diff
@@ -131,13 +132,17 @@ abstract class WholeNumberBinnedArray[T](length: Int, bounds: (T, T)) extends Bi
   private val min = convertToLong(bounds._1)
   private val max = convertToLong(bounds._2)
 
-  require(min < max,
-    s"Upper bound must be greater than lower bound: lower='${bounds._1}'($min) upper='${bounds._2}'($max)")
+  if (min > max) {
+    val msg = s"Upper bound must be greater than lower bound: lower='${bounds._1}'($min) upper='${bounds._2}'($max)"
+    logger.error(msg)
+    throw new IllegalArgumentException(msg)
```

Comment:
aren't these are still going to spam the console? now it will just log as well. (`require` throws an IllegalArgumentException)